WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -287,7 +287,12 @@ private extension SiteStatsPeriodViewModel {
             mostRecentChartData.periodEndDate == periodSummary.periodEndDate {
             self.mostRecentChartData = periodSummary
         } else if let periodSummary = periodSummary, let chartData = mostRecentChartData, periodSummary.periodEndDate > chartData.periodEndDate {
```

Comment:
I wonder if these checks can be simplified or if some of them are not needed:

The last one set `chartData` to `mostRecentChartData` and then set `mostRecentChartData` to this `chartData`

```
else if <...> let chartData = mostRecentChartData <...> {
    mostRecentChartData = chartData
}
```

Reply:
@staskus I had thought this myself but I didn't want to introduce a bug + not sure what the intention was so I took your first suggestion and only did comments for now